cfg/builder/builder_walk.cc

```diff
@@ -218,12 +218,15 @@ BasicBlock *CFGBuilder::walk(CFGContext cctx, ast::Expression *what, BasicBlock
             },
             [&](ast::UnresolvedConstantLit *a) { Exception::raise("Should have been eliminated by namer/resolver"); },
             [&](ast::ConstantLit *a) {
+                auto aliasName = cctx.newTemporary(core::Names::cfgAlias());
                 if (a->symbol == core::Symbols::StubModule()) {
-                    current->exprs.emplace_back(cctx.target, a->loc, make_unique<Alias>(core::Symbols::untyped()));
+                    current->exprs.emplace_back(aliasName, a->loc, make_unique<Alias>(core::Symbols::untyped()));
```

Comment:
sythesizeExpr?

Reply:
Doesn't fix the problems with LSP, unfortunately :(